### ISSUE TITLE
Multi-line selection key combination gets rebound on Linux

### ISSUE THREAD

**Issue by GeckoEidechse** [yzhang-gh/vscode-markdown#857]
## Problem

When editing a Markdown document using this extension the key-combination to select multiple lines on Linux is rebound from `Shift`+`Alt`+`Arrows` to `Shift`+`Ctrl`+`Arrows`. This seems to only happen on markdown documents.

Disabling the extension and reloading vscode resulted in the key combination being `Shift`+`Alt`+`Arrows` again for markdown documents.

Extension version is `v3.4.0`

vscode version:

```
Version: 1.51.1
Commit: e5a624b788d92b8d34d1392e4c4d9789406efe8f
Date: 2020-11-10T23:31:29.624Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 5.4.0-54-generic snap
```

## How to reproduce

Install the extension and open a markdown document on Linux.`Shift`+`Alt`+`Arrows` no longer works as expected on Linux and rather pastes the previous line again, similar to the behaviour it has on Windows.

## Error message in the console
N/A

**Comment by yzhang-gh** [comment]
Thanks for the feedback.

I'm a little surprised that on Linux this keybinding has a different meaning from that on Windows and Mac, which is "copy line up/down". (see a full reference [here](https://code.visualstudio.com/docs/getstarted/keybindings#_keyboard-shortcuts-reference))

It should be fixed in the [dev build](https://github.com/yzhang-gh/vscode-markdown#latest-development-build). Please check it out.

**Comment by GeckoEidechse** [comment]
Can confirm that it's fixed for me with 461193ce5be4329c12ec07cd4d7bac2fb89b7f1d. Thanks :D
